src/ActivityStickyQueues/StickyActivities.cs

```diff
@@ -0,0 +1,49 @@
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Temporalio.Activities;
+
+namespace TemporalioSamples.ActivityStickyQueues;
+
+public class StickyActivities
```

Comment:
Probably want to make this a static class w/ static methods if there is no state